docs/state-of-progressive-decentralization.md

```diff
@@ -17,70 +17,38 @@ import DraftExpectationsPartial from '@site/docs/partials/\_draft-expectations-p
+| **Arbitrum&nbsp;One**  | **Permissionless**           | Validation on Arbitrum One now follows the BoLD protocol, any node implementing the BoLD protocol can join. You can learn more about BoLD in this [introduction to BoLD](https://docs.arbitrum.io/how-arbitrum-works/bold/gentle-introduction) |
+| **Arbitrum&nbsp;Nova** | **Permissioned**             | Arbitrum Nova remains permissioned as its Data Availability committee is a trusted one                                                                                                                                                         |
 
-:::
-
-- **Description**: AnyTrust chains like Arbitrum Nova rely on a permissioned committee to store the chain's data and provide it on demand.
-- **Current status**: 6-member committee. The Arbitrum Nova chain has a 6-party DAC, whose members can be seen [here](#data-availability-committee-members). Governance has the ability to remove or add members to the committee.
-- **Risks**: If 5 of the 6 committee members in conjunction with the Sequencer behave maliciously and collude, the safety of the system can be compromised.
-- **Changes to Current status**: The Arbitrum governance system (see #1) currently has the power to change the DAC, such as by adding or removing members or modifying the power it has over the system.
+- **Risks**: If there is not a single honest active validator, and a malicious validator proposes an invalid state update, the system's safety could be compromised.
 
 ### Allowlisted validators
 
 import { AddressExplorerLink as AEL } from '@site/src/components/AddressExplorerLink'
 
-These are the current [allowlisted validators](#2-validator-ownership) for every public Arbitrum chain:
-
-#### Arbitrum One
-
-- <AEL address={"0x0fF813f6BD577c3D1cDbE435baC0621BE6aE34B4"} chainID={1} />
-- <AEL address={"0x54c0D3d6C101580dB3be8763A2aE2c6bb9dc840c"} chainID={1} />
-- <AEL address={"0x56D83349c2B8DCF74d7E92D5b6B33d0BADD52D78"} chainID={1} />
-- <AEL address={"0x610Aa279989F440820e14248BD3879B148717974"} chainID={1} />
-- <AEL address={"0x6Fb914de4653eC5592B7c15F4d9466Cbd03F2104"} chainID={1} />
-- <AEL address={"0x758C6bB08B3ea5889B5cddbdeF9A45b3a983c398"} chainID={1} />
-- <AEL address={"0x7CF3d537733F6Ba4183A833c9B021265716cE9d0"} chainID={1} />
-- <AEL address={"0x83215480dB2C6A7E56f9E99EF93AB9B36F8A3DD5"} chainID={1} />
-- <AEL address={"0xAB1A39332e934300eBCc57B5f95cA90631a347FF"} chainID={1} />
-- <AEL address={"0xB0CB1384e3f4a9a9b2447e39b05e10631E1D34B0"} chainID={1} />
-- <AEL address={"0xF8D3E1cF58386c92B27710C6a0D8A54c76BC6ab5"} chainID={1} />
-- <AEL address={"0xdDf2F71Ab206C0138A8eceEb54386567D5abF01E"} chainID={1} />
-- <AEL address={"0xf59caf75e8A4bFBA4e6e07aD86C7E498E4d2519b"} chainID={1} />
+These are the current [allowlisted validators](#2-validator-ownership) for Arbitrum One:
```

Comment:
I'm not sure this makes sense to still be here given that there isn't an allowlist any more.

Reply:
Thanks for the nudge, @eljobe. I've removed them. Care to approve?